arc_cache.go

```diff
@@ -19,7 +20,7 @@ type arccache struct {
 	blockstore Blockstore
 }
 
-func newARCCachedBS(ctx context.Context, bs Blockstore, lruSize int) (*arccache, error) {
+func newARCCachedBS(ctx context.Context, bs ib.Blockstore, lruSize int) (*arccache, error) {
```

Comment:
There is no need to use ib for Blockstore since the type is already aliased.

Reply:
good point